consensus/hotstuff/notifications/telemetry.go

```diff
@@ -25,20 +25,25 @@ import (
 //     -- a local timeout has been initiated
 //   - Each path through the state machine is identified by a unique id.
 //
+// Additionally, the TelemetryConsumer reports events related to vote and timeout aggregation
+// but those events are not bound to a path, so they are reported differently.
 // Generally, the TelemetryConsumer could export the collected data to a variety of backends.
 // For now, we export the data to a logger.
 //
 // Telemetry does NOT capture slashing notifications
 type TelemetryConsumer struct {
 	NoopConsumer
-	pathHandler *PathHandler
+	pathHandler  *PathHandler
+	noPathLogger zerolog.Logger
 }
 
 var _ hotstuff.Consumer = (*TelemetryConsumer)(nil)
 
-func NewTelemetryConsumer(log zerolog.Logger, chain flow.ChainID) *TelemetryConsumer {
+func NewTelemetryConsumer(log zerolog.Logger) *TelemetryConsumer {
```

Comment:
The telemetry consumer is of limited use without the logger being tagged with the `chain_id` we're reporting about. That used to be ensured here by passing it in explicitly - now we are relying on the caller to make sure the logger argument is tagged appropriately.

**Suggestion:** Add a note in the godoc here and in `NewPathHandler` that `log` must have a field called `chain` set to the chain ID of the consensus instance.

Reply:
Do we really need it for main consensus? 
I was under impression that `chain_id` really matters for cluster consensus for us to clearly see what log messages are for which epoch, but with main consensus it should be "mainnet" every-time? This is the main reason why I didn't add `chain_id` for main consensus events but applied it for cluster consensus.

Reply:
Eventually consensus nodes will need to run Follower instances for cluster consensus. Then it would be useful to be explicit about which chain we're referring to in logs, since consensus nodes would be worried about multiple chain IDs at the same time. I agree it's not strictly necessary now, but it doesn't hurt and then we don't need to change it later.